pkgs/by-name/im/imager/package.nix

```
@@ -0,0 +1,112 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  gtk2-x11,
+  pkg-config,
+  python3,
+  gfortran,
+  lesstif,
+  cfitsio,
+  getopt,
+  perl,
+  groff,
+  which,
+  darwin,
+  ncurses,
+}:
+
+let
+  python3Env = python3.withPackages (
+    ps: with ps; [
+      numpy
+      setuptools
+    ]
+  );
+in
+
+stdenv.mkDerivation rec {
```

Use the finalAttrs pattern if possible 
```suggestion
stdenv.mkDerivation (finalAttrs: {
```